rskj-core/src/test/java/co/rsk/net/light/LightSyncProcessorTest.java

```diff
@@ -141,29 +141,12 @@ public void processStatusMessageAndShouldAskForAndReceiveBlockHeaderCorrectly()
 
         BlockHeadersMessage blockHeadersMessage = new BlockHeadersMessage(requestId, bHs);
 
-        lightClientHandler.channelRead0(ctx, blockHeadersMessage);
+        lightMessageHandler.processMessage(lightPeer, blockHeadersMessage, ctx, lightClientHandler);
 
         assertEquals(1, lightPeer.getBlocks().size());
         assertEquals(bHs, lightPeer.getBlocks());
     }
 
-    @Test
```

Comment:
Ok, we should implement a peer handler before accepting messages from more than one peer, we are not doing anything to peers with bad behavior.

Reply:
This is a mistake on my part, this should not be removed, it relates to a constant I wrongly changed